_config.yml

```diff
@@ -258,9 +258,9 @@ kramdown:
 compress_html:
   comments:            ["<!-- ", " -->"]
   clippings:           all
-  endings:             all
-  ignore:
-    envs:              [development]
+  endings:             ""
+ # ignore:
+ #   envs:              [development]
```

Comment:
Can we delete 262-263? I prefer not to leave dead code commented out.

Reply:
Absolutely, that is removed now in 3265262